src/test/java/org/takes/rq/RqLengthAwareTest.java

```diff
@@ -81,4 +83,89 @@ public void addsBigLengthToBody() throws IOException {
         );
     }
 
+    /**
+     * RqLengthAware can read byte.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void readsByte() throws IOException {
+        final String data = "{test}";
+        final InputStream stream = new RqLengthAware(
+            new RqFake(
+                Arrays.asList(
+                    "GET /test1",
+                    "Host: b.example.com",
+                    "Content-type: text/json"
```

Comment:
@longtimeago According  the HTTP 1.1 specification any message containing an entity-body should include a Content-Type header. But  we can omit it here.  You're right